.github/workflows/inspektor-gadget.yml

```diff
@@ -170,6 +172,9 @@ jobs:
       # ALERT This action must be run after code was checkout otherwise it will
       # not find this file.
       uses: ./.github/actions/install-debian-packages
+    - uses: github/codeql-action/init@v2
+      with:
+        languages: go, cpp
```

Comment:
Why cpp?

Reply:
`codeql` summarizes C and C++ into one category: C++ https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/

Ofc the question is still if it even can understand the our headers and ebpf source at all

Reply:
Yeah, I don't think it'll provide anything related to bpf. I agree that we can merge and check what happens.